src/commands/utility/sub/username/add.ts

```diff
@@ -0,0 +1,44 @@
+import type { CommandInteraction } from 'discord.js';
+import i18next from 'i18next';
+import type { Redis } from 'ioredis';
+import { addBannedUsername, type RawBannedUsernameData } from '../../../../util/bannedUsernames.js';
+
+export async function add(
+	interaction: CommandInteraction<'cached'>,
+	data: RawBannedUsernameData,
+	redis: Redis,
+	locale: string,
+): Promise<void> {
+	/**
+	 * @param input `/(hype)\s+(events?|messages?|apply|team|system)/i`
+	 * @returns `(hype)\s+(events?|messages?|apply|team|system)`
+	 */
+	const parseRegex = (input: string) => {
```

Comment:
I don't know if I should put this on a different file

Reply:
Yeah probably in the util folder 👍